pkg/process/config/config_test.go

```diff
@@ -225,6 +278,16 @@ func TestDefaultConfig(t *testing.T) {
 	assert.Equal(6062, agentConfig.ProcessExpVarPort)
 
 	os.Unsetenv("DOCKER_DD_AGENT")
+
+	assert.Equal("info", config.Datadog.GetString("log_level"))
+	assert.True(config.Datadog.GetBool("log_to_console"))
+	assert.Equal(config.DefaultProcessAgentLogFile, config.Datadog.GetString("process_config.log_file"))
+	assert.Equal(config.DefaultDDAgentBin, config.Datadog.GetString("process_config.dd_agent_bin"))
+	assert.Equal(config.DefaultGRPCConnectionTimeoutSecs, config.Datadog.GetInt("process_config.grpc_connection_timeout_secs"))
+	assert.True(config.Datadog.GetBool("process_config.remote_tagger"))
+	assert.False(config.Datadog.GetBool("process_config.process_discovery.enabled"))
+	assert.Equal(4*time.Hour, config.Datadog.GetDuration("process_config.process_discovery.interval"))
+
```

Comment:
You don't need to test some of these here (not part of `AgentConfig`.

Reply:
Good advice!